src/families/stellar/bridge/libcore.js

```diff
@@ -0,0 +1,265 @@
+// @flow
+import invariant from "invariant";
+import { BigNumber } from "bignumber.js";
+import {
+  AmountRequired,
+  NotEnoughBalance,
+  FeeNotLoaded,
+  InvalidAddressBecauseDestinationIsAlsoSource
+} from "@ledgerhq/errors";
+import {
+  StellarWrongMemoFormat,
+  StellarMinimumBalanceWarning,
+  StellarNewAccountMinimumTransaction,
+  StellarSourceHasMultiSign
+} from "../../../errors";
+import { validateRecipient } from "../../../bridge/shared";
+import type { AccountBridge, CurrencyBridge } from "../../../types";
+import type { Transaction } from "../types";
+import { scanAccounts } from "../../../libcore/scanAccounts";
+import { getAccountNetworkInfo } from "../../../libcore/getAccountNetworkInfo";
+import { sync } from "../../../libcore/syncAccount";
+import broadcast from "../libcore-broadcast";
+import signOperation from "../libcore-signOperation";
+import { withLibcore } from "../../../libcore/access";
+import memoTypeCheck from "../memo-type-check";
+
+import { getWalletName } from "../../../account";
+import { getOrCreateWallet } from "../../../libcore/getOrCreateWallet";
+import { getCoreAccount } from "../../../libcore/getCoreAccount";
+
+const checkRecipientExist = (account, recipient) =>
```

Comment:
we ?probably should memoize it (lru cache, small time like 5mn max memory – maybe it's already the default?). but also clear the memo when you sign the operation (because recipient can become existing after that?)